src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/ViewProviders/IDependenciesGraphViewProvider.cs

```diff
@@ -60,6 +60,6 @@ internal interface IDependenciesGraphViewProvider
         bool MatchSearchResults(
             IDependency topLevelDependency,
             Dictionary<string, HashSet<IDependency>> searchResultsPerContext,
-            [NotNullWhenTrue] out HashSet<IDependency>? topLevelDependencyMatches);
+            [MaybeNullWhen(false)] out HashSet<IDependency> topLevelDependencyMatches);
```

Comment:
If it "may be" null, then why isn't it marked nullable?

Reply:
It actuall was until my force push. You use annotations when you want to override the declared type's nullability. Another example would be a `string?` property where the setter cannot be null (i.e. annotate the setter with `[DisallowNull]`). It's also useful on generic types such as `IEquatable<T>` (which reminds me, I should update one of the annotations for that interface in the dependencies node.)